api/src/main/java/com/datastrato/gravitino/rel/TableChange.java

```diff
@@ -581,8 +581,11 @@ final class After implements ColumnPosition {
     private final String column;
 
     private After(String column) {
-      assert column != null;
```

Comment:
if (null == column) {
	throw new illlegalArumentExcepiotin("xxx");
}

this.column = column.

Reply:
got it, and i have updated it. Thank you for reviewing @yuqi1129